src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/dcoff.hpp

```diff
@@ -21,6 +22,15 @@ enum class DCOffMode : int {
     CAST_SCALE,
 };
 
+namespace pattern_utils {
+
+std::shared_ptr<ov::op::v0::MatMul> find_matmul_downwards(const std::shared_ptr<ov::Node>& start_node);
+std::shared_ptr<ov::op::v0::MatMul> get_root_matmul(ov::pass::pattern::Matcher& m);
+bool transpose_required(const std::shared_ptr<ov::op::v0::MatMul>& matmul_node);
+ov::Tensor transpose_tensor(const ov::Tensor& tensor);
+
+}  // namespace matmul_utils
+
```

Comment:
you don't need these definitions here if there's just `dcoff.cpp` is using it.

Reply:
Removed!